src/views/Farm/Farm.tsx

```diff
@@ -19,17 +20,19 @@ const Farm = (props: { title: string }) => {
   return (
     <Page>
       <Container>
-        <StyledPageHeader data-cy='liquidity-mining-title'>Liquidity Mining Programs</StyledPageHeader>
+        <StyledPageHeader data-cy='liquidity-mining-title'>
+          Liquidity Mining Programs
+        </StyledPageHeader>
         <Spacer size='sm' />
         <StyledPageSubheader data-cy='liquidity-mining-subtitle'>
           Earn rewards for supplying liquidity for Index Coop products
         </StyledPageSubheader>
         <Spacer />
       </Container>
-      <Treasury data-cy="treasury"/>
+      <Treasury data-cy='treasury' />
       <Spacer />
       <Container>
-        <V3FarmCard />
+        <V3FarmCard farm={DpiEthRewards} />
```

Comment:
So when we want to use this in the future we just do `<V3FarmCard farm={mviEthRewards}/>`?